src/planner/expr.go

```diff
@@ -267,8 +267,6 @@ func parserSelectExpr(expr *sqlparser.AliasedExpr, tbInfos map[string]*TableInfo
 			if tableName == "" {
 				if len(tbInfos) == 1 {
 					tableName, _ = getOneTableInfo(tbInfos)
-				} else {
```

Comment:
`create table t1(a int primary key);`
`create table t2(a int primary key);`
`select a from t1 join t2 on t1.a=t2.b;`
The error will reported from MySQL instand of Radon.
If execute `select a from t1 join t2 on t1.a=t2.b;`, we'll get no error info.

Reply:
This problem and the next are resolved by adding a additional check.